### PR TITLE
fix: /issues/117; add new cases into Property.robot and Combobox.robot

### PR DESCRIPTION
**Pull request by JimRevolutionist**
@Nepitwin   pls review
I have added new cases and executed command '.\keen.bat test'. Below is my results.

image

### CODE REVIEW

**Comment by Nepitwin** [comment]
Looks good for now comment can be deleted or you can merge it ;)

**Comment by Nepitwin** [comment]
Test looks asweel all fine for all python versions

File: src/FlaUILibrary/flaui/module/property.py [hunk 1]
```diff
@@ -41,6 +41,9 @@ class Action(Enum):
         IS_TEXT_PATTERN_SUPPORTED = "IS_TEXT_PATTERN_SUPPORTED"
         IS_TOGGLE_PATTERN_SUPPORTED = "IS_TOGGLE_PATTERN_SUPPORTED"
         IS_VALUE_PATTERN_SUPPORTED = "IS_VALUE_PATTERN_SUPPORTED"
+        # for ExpandCollapsePattern
```
[the remark this answers is not in the document]
Comment (JimRevolutionist) [reply src/FlaUILibrary/flaui/module/property.py:44]: Okay, I see.

**Comment by Nepitwin** [comment]
@JimRevolutionist if you want you can merge it or i will do it  if its all fine for you.

**Comment by JimRevolutionist** [comment]
Aha, you should take a hand to merge this issue. @Nepitwin 
By the way, I did not have authority to merge pr or edit a issue's properties, like Labels, Assignees.

**Comment by Nepitwin** [comment]
@JimRevolutionist 

Thanks for the info you are right. I updated all

**Pull request merged**